### DISCUSSION TITLE
服务器系统行么

### DISCUSSION THREAD

**Discussion by LotusRootMonkey** [chatchat-space/Langchain-Chatchat#3566] [General]
你好，请问如果我想部署这个项目，但是我想用服务器操作系统，比如ubuntu-22.04.4-live-server-amd64.iso，加上tesla的计算卡，能成功部署么，还是说，一定要用有桌面的操作系统，比如ubuntu-22.04.4-desktop-amd64.iso

**Comment by zRzRzRzRzRzRzR** [comment]
可以，默认就是ubuntu
